tests/unit/api/PacbioRun.spec.js

```diff
@@ -3,6 +3,7 @@ import Response from '@/api/Response'
 import * as Run from '@/api/PacbioRun'
 import build from '@/api/ApiBuilder'
 import Api from '@/api'
+import { it } from 'vitest'
```

Comment:
Ahh, VScode keeps doing this, harmless, but ultimately unnecessary as these are available globally.  Still, makes me wonder if we should just import them explicitly the whole time.

Reply:
Generally explicit beats implicit and global, I would have thought.